isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -82,7 +82,7 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
          and $ISIS3TESTDATA=$CONDA_PREFIX/testdata.  The user can specify different directories
          for both of these optional values:
 
-         <strong>./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
+         <strong>python $CONDA_PREFIX/scripts/isis3VarInit.py --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
```

Comment:
The script has a shebang, so you don't need python at the start

Reply:
For my conda env, the scripts are all downloaded with 664 permissions set, so I can't actually run the script without using the python interpreter

Reply:
:+1: